cmd/server/assets/codestatus/show.html

```diff
@@ -18,45 +18,88 @@ <h1>Verification code status</h1>
       Status of a verification code previously shared with your patient.
     </p>
 
-    <div class="card mb-3">
+    <div class="card mb-3" style="max-width: 30rem">
       <div class="card-header">
         Code status
       </div>
-      <div class="card-body">
-        <div class="list-group">
+      <div class="list-group">
+        <div class="list-group-item">
           <h5 class="mb-1">UUID</h5>
           <p class="mb-1 text-monospace">{{.uuid}}</p>
         </div>
-        <div class="list-group">
+        <div class="list-group-item">
           <h5 class="mb-1">Status</h5>
           <p class="mb-1">{{.status}}</p>
         </div>
-        <div class="list-group">
+        <div class="list-group-item">
           <h5 class="mb-1">Expiry</h5>
           <span id="code-expires-at" class="sm text-danger"></span>
         </div>
       </div>
+      <button type="button" id="invalidate" class="btn btn-danger btn-sm">Invalidate code now</button>
     </div>
 
     <div>
-      <p>
-        <a href="/code/status">&larr; Enter another code</a>
-      </p>
+      <a href="/code/status">&larr; Enter another code</a>
     </div>
   </main>
 
   {{template "scripts" .}}
   {{template "codescripts" .}}
 
   <script type="text/javascript">
+    let $buttonInvalidate = $('button#invalidate');
     let $codeExpiresAt;
     let expires = {{.expires }};
+    let uuid = {{.uuid }}
```

Comment:
sc

Reply:
switching between go and javascript is killin me